src/BootstrapManager.php

```diff
@@ -71,6 +71,7 @@ public static function getInstance() {
 
 	/**
 	 * @since  1.0
+	 * @suppress PhanTypeMismatchProperty
```

Comment:
Would it be better to just set |null on both $moduleDefinition and $instance as they are set as null by default etc?